region/maridia/outer.json

```diff
@@ -3277,6 +3277,52 @@
                     "HiJump"
                   ]
                 },
+                {
+                  "name": "Cross Room Jump to the Frozen Fish",
+                  "notable": false,
+                  "requires": [
+                    "HiJump",
+                    "canCrossRoomJumpIntoWater",
+                    "canMomentumConservingTurnaround",
+                    "canTrickyUseFrozenEnemies",
+                    "canTrickyJump",
+                    "canDownGrab",
+                    {"or": [
+                      {"and": [
+                        "Wave",
+                        "Spazer"
+                      ]},
+                      "Plasma"
```

Comment:
can a charge shot help?

Reply:
I was trying missiles to make ice only work and couldnt get it.  Charge+Ice is going to be too slow.  But Spazer or wave on their own with ice works so long as you shoot the fish in a particular way.